src/components/Slider/Slider.js

```diff
@@ -35,19 +33,37 @@ class Slider extends React.Component {
 		this.state = {
 			slides: HELP_SLIDES,
 			speed: 100,
+			articles: props.articles || [],
 			autoReel: false
 		};
 	}
 	componentDidMount() {
```

Comment:
the similar comments here... + move it to the separate method... and just invoke it here